ds-dash/files/docker/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+  fi
+fi
+
+cd src && exec python3 app.py
```

Comment:
Why not start it with gunicorn directly? Since dash is based on flask, we face the same issues as with the other quickstarters

Reply:
I do not see this quickstarter something that is going to be a production ready app, but more a fast prototyping tool for fast interactions and testings with users.

Reply:
Does it have any downside using gunicorn? Because I definitely see and already have use-cases where dash could/will be used "productive"

Reply:
let's meet and define what does here mean "productive"